src/otx/core/types/explain.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+"""OTX explain type definition."""
+
+from __future__ import annotations
+
+from enum import Enum
+
+
+class TargetExplainGroup(str, Enum):
```

Comment:
Could you explain what is the difference between `IMAGE` and `PREDICTIONS` ?

Reply:
I agree with @sungmanc. Would it be possible to expand the docstring a bit? A user may not be familiar with this concept, and it would be good if we make this clear for every level of users

Reply:
Extended docstring